app/connectors/GetBusinessDetailsConnector.scala

```diff
@@ -47,29 +48,33 @@ class GetBusinessDetailsConnector @Inject()(val http: HttpClient,
 
     logger.debug("" +
       s"Calling GET $url \n\nHeaders: $headerCarrier \nAuth Headers: $headers")
-    http.GET[HttpResponse](url = url, headers = headers)(httpReads, headerCarrier, implicitly) map {
-      response =>
-        response.status match {
-          case OK =>
-            logger.debug(s"RESPONSE status:${response.status}, body:${response.body}")
-            response.json.validate[IncomeSourceDetailsModel](jsonReads).fold(
-              invalid => {
-                logger.error(s"Validation Errors: $invalid")
-                IncomeSourceDetailsError(Status.INTERNAL_SERVER_ERROR, "Json Validation Error. Parsing Business Details")
-              },
-              valid => {
-                logger.info("successfully parsed response to getBusinessDetails")
-                valid
-              }
-            )
-          case NOT_FOUND =>
-            logger.warn(s" RESPONSE status: ${response.status}, body: ${response.body}")
-            IncomeSourceDetailsNotFound(response.status, response.body)
-          case _ =>
-            logger.error(s"RESPONSE status: ${response.status}, body: ${response.body}")
-            IncomeSourceDetailsError(response.status, response.body)
-        }
-    } recover {
+    http
+      .get(url"$url")
+      .setHeader(headers.head)
```

Comment:
what about rest of the headers in the _headers_ value? I would assume we need these as well